common/djangoapps/student/views.py

```diff
@@ -892,6 +883,35 @@ def dashboard(request):
     return response
 
 
+@login_required
+def course_run_refund_status(request, course_id):
+    """
+    Get Refundable status for a course.
+
+    Arguments:
+        request: The request object.
+        course_id (str): The unique identifier for the course.
+
+    Returns:
+        Json response.
+
+    """
+
+    try:
+        course_key = CourseKey.from_string(course_id)
+        course_enrollment = CourseEnrollment.get_enrollment(request.user, course_key)
+
+    except InvalidKeyError:
+        logging.exception("InvalidKeyError has occured while getting refund status for a course.")
```

Comment:
#nitpick Can we make this message more human understandable like `The course key used to get refund status caused InvalidKeyError during look up`?